packages/compass-e2e-tests/helpers/commands/set-connect-form-state.ts

```diff
@@ -24,13 +25,33 @@ async function waitForElementAnimations(browser: CompassBrowser, element: any) {
   });
 }
 
+const colorMap: Record<string, string> = {
+  'no-color': 'No Color',
+  color1: 'Red',
+  color2: 'Pink',
+  color3: 'Orange',
+  color4: 'Yellow',
+  color5: 'Green',
+  color6: 'Teal',
+  color7: 'Blue',
+  color8: 'Iris',
+  color9: 'Purple',
+};
+
+function colorValueToName(color: string): string {
```

Comment:
There's probably a better way to do this, but this is so we can select an option by its display value. ie. the select's value is 'color1', but the option is displayed as 'Red'. We can probably import this from somewhere rather than hardcoding it..